src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
+  const DAILY_RENT = 40;
+  const MEDIUM_TERM = 3;
+  const REG_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const BIG_DISCOUNT = 50;
+
+  totalCost = DAILY_RENT * days;
+
+  if (days >= MEDIUM_TERM && days < LONG_TERM) {
+    return totalCost - REG_DISCOUNT;
+  }
+
```

Comment:
can be simplify
```suggestion
  if (days >= MEDIUM_TERM) {
    return totalCost - REG_DISCOUNT;
  }

```

Reply:
We would need to swap the if constructions around, as 7 is higher than MEDIUM_TERM. We would need to check for long term, then medium term. But you are still correct, it is a retention from when I did not have return inside the if constructions. So yes, it can be simplified further.